scripts/bump-versions.sh

```diff
@@ -3,7 +3,7 @@
 set -ex
 
 #
-# To use this function, make sure you have maven installed, and run `cargo install cargo-edit` first
+# Run `cargo install cargo-edit` once before you use this function
```

Comment:
I like to be more helpful than this by checking for missing tools in the code, e.g.:

```
if ! (cargo set-version --version) >/dev/null 2>&1; then
    echo >&2 "$0: cargo-edit not installed, please run 'cargo install cargo-edit'"
    exit 1
fi
```